lib/hyperreduction/S_OPT.C

```diff
@@ -0,0 +1,531 @@
+/******************************************************************************
+ *
+ * Copyright (c) 2013-2021, Lawrence Livermore National Security, LLC
+ * and other libROM project developers. See the top-level COPYRIGHT
+ * file for details.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR MIT)
+ *
+ *****************************************************************************/
+
+// Description: Interface to the S_OPT algorithm to determine the rows of the
+// rhs to be sampled for the interpolation of the rhs->
```

Comment:
It's not clear what `rhs->` is (in a few comments in this file). Is it a typo?

Reply:
Hmm. Yes, I am not sure where this came from.